src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -259,11 +131,19 @@
   </div>
 </dl>
 
+<ng-container *ngIf="activeRecord.vehicleType === 'hgv'">
+  <vtm-vehicle-summary-hgv [activeRecord]="activeRecord"></vtm-vehicle-summary-hgv>
+</ng-container>
+
 <ng-container *ngIf="activeRecord.vehicleType === 'psv'">
   <vtm-vehicle-summary-psv [activeRecord]="activeRecord"></vtm-vehicle-summary-psv>
 </ng-container>
 
-<dl class="govuk-summary-list">
+<ng-container *ngIf="activeRecord.vehicleType === 'trl'">
+  <vtm-vehicle-summary-trl [activeRecord]="activeRecord"></vtm-vehicle-summary-trl>
+</ng-container>
+
```

Comment:
we should also have tests to cover these child components from the parents perspective. I can guide here